Clickstream/Core/Interface/ClickStream.swift

```diff
@@ -179,3 +179,33 @@ extension Clickstream {
     }
 }
 #endif
+
+// MARK: - Code below here is support for the Clickstream's EventVisualizer.
+#if EVENT_VISUALIZER_ENABLED
+extension Clickstream {
+    
+    /// Initialise event visualizer state tracking
+    /// - Parameters:
+    ///   - guid:String
+    ///   - eventTimestamp:String
+    ///   - storageGuid:String
+    ///   - storageEventTimestamp:String
+    public func setEventVisualizerStateTracking(guid: String? = nil,
```

Comment:
What's the difference between guid and storageGuid? Why do we need storageGuid and storageEventTimestamp?

Reply:
These are optional, so if the client does not provide anything what output they will see?